docs/learn/learn-async-backing.md

```diff
@@ -7,104 +7,312 @@ keywords: [parachains, backing, parablock, perspective parachains, unincluded se
 slug: ../learn-async-backing
 ---
 
-Asynchronous backing is a feature that introduces
-[pipelining](https://www.techtarget.com/whatis/definition/pipelining) to the parachain block
-generation and validation process. It is analogous to the logical pipelining of processor
-instruction in "traditional" architectures, where some instructions may be executed before others
-are complete. Instructions may also be executed in parallel, enabling multiple parts of the
-processor to be working on potentially different instructions at the same time.
+import RPC from "./../../components/RPC-Connection";
 
-Bundles of state transitions represented as blocks may be processed similarly. In the context of
-Polkadot, it aims to increase the throughput of the entire network.
+:::info Learn about Parachain Consensus
+
+To fully follow the material on this page, it is recommended to be familiar with the primary stages
+of the [Parachain Protocol](./learn-parachains-protocol.md).
+
+:::
+
+## Synchronous Backing
+
+Before diving into asynchronous backing, it is important to understand what synchronous backing is
+and what its main limitations are.
 
-:::info
+:::info What is backing?
 
-In order to realize which exact processes this upgrade improves, it is recommended to be familiar
-with the stages of parablock validation, which you may [read here](./learn-parachains-protocol.md).
+**Backing** refers to the process in which a parablock is verified by a subset of validators who
+determine the block’s validity. It is an important step in the validation process for parablocks, as
+it is the first line of defense in ensuring valid state transitions. Validators who back the
+parablock put their stake at risk, and if the block is later found invalid, those validators are
+slashed.
 
 :::
 
-Asynchronous backing enables logical pipelining over the parablock
-[**generation**](https://wiki.polkadot.network/docs/learn-parachains-protocol#collators),
-[**backing**](https://wiki.polkadot.network/docs/learn-parachains-protocol#parachain-phase), and
-[**inclusion**](https://wiki.polkadot.network/docs/learn-parachains-protocol#inclusion-pipeline)
-processes. A parablock may be at different stages, but multiple parablocks can be processed
-simultaneously (in parallel) if needed. Most notably, parablock N + 1 can be generated and backed
-while its predecessor, parablock N, is undergoing inclusion on the relay chain. Processes can occur
-while their ancestors are included on the relay chain.
+In synchronous backing, parablock generation is tightly coupled to the relay chain's progression on
+a one-to-one basis. Every parablock must be generated and backed within a relay-chain block
+(six-second window), and (if successfully backed) it will be included in a relay-chain block (often
+referred to as the **relay parent** as the parablock anchors itself to it) after an additional six
+seconds. Thus, a parablock can be produced every 12 seconds because a new parablock can be produced
```

Comment:
I would mention that inclusion means other validators have attested to having received erasure coded chunks of the parablock data. You don't need to go in depth about this, just link to another part of the wiki that covers availability. The point is that this is why inclusion takes two relay chain blocks. Then from hereon out you can discuss backing and inclusion as one step to simplify your explanatoin.